File: 1-two-sum/1-two-sum.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(nums, target) {
    for (i = 0; i < nums.length; i++) {
        for (n = 0; n < nums.length; n++) {
            if (i === n) {
                continue
            }
            else if (parseInt(nums[i] + nums[n]) === target) {
                let targetPair = [i,n];
                return targetPair;
            }
            else {
                continue
            }
        }
    }
};

File: 1-two-sum/1-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        
        for n in nums:
            if target - n in nums[nums.index(n)+1:]:
                return [nums.index(n),len(nums)-(nums[::-1].index(target-n))-1]
        
        return [69]